tooling/linux_repro_build_compare.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/sh
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# This script examines the given SBOM metadata file, and then builds the exact same binary
+# and then compares with the Temurin JDK for the same build version, or the optionally supplied TARBALL_URL.
+
+[ $# -lt 1 ] && echo Usage: $0 SBOMURL TARBALL_URL && exit 1
+SBOMURL=$1
+TARBALL_URL=$2
+ANT_VERSION=1.10.5
+ANT_CONTRIB_VERSION=1.0b3
+
+installPrereqs() {
```

Comment:
A couple of lovely functions, can we have more of those in this script?

Reply:
added